hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/controller/ContractController.java

```diff
@@ -154,6 +157,13 @@ private Mono<GenericErrorResponse> invalidTxnBodyError(final ServerWebInputExcep
         return errorResponse(e.getReason(), e.getMostSpecificCause().getMessage(), StringUtils.EMPTY);
     }
 
+    @ExceptionHandler
+    @ResponseStatus(NOT_FOUND)
+    private Mono<GenericErrorResponse> tokeOrContractMissingError(final EntityNotFoundException e) {
+        log.warn("Missing entity error: {}", e.getMessage());
```

Comment:
May expand to other things besides entities.
```suggestion
        log.warn("Not found: {}", e.getMessage());
```